todocore/src/main/java/com/knighttodo/todocore/service/DayTodoService.java

```diff
@@ -90,5 +90,17 @@ public DayTodoVO updateIsReady(UUID dayId, UUID dayTodoId, boolean isReady) {
         dayTodoVO = dayTodoMapper.toTodoVO(dayTodoRepository.save(dayTodoMapper.toTodo(dayTodoVO)));
         return experienceService.calculateExperience(dayTodoVO);
     }
+
+    @Transactional
+    public DayTodoVO updateDayTodoById(UUID dayTodoId, Integer newDayTodoOrderNumber) {
+        DayTodoVO dayTodoVO = findById(dayTodoId);
+        dayTodoVO.setOrderNumber(newDayTodoOrderNumber);
+        DayTodo dayTodo = dayTodoRepository.save(dayTodoMapper.toTodo(dayTodoVO));
+        return dayTodoMapper.toTodoVO(dayTodo);
+    }
+
+    public boolean isExistDayTodo(UUID dayTodoId) {
```

Comment:
this method is redundant, you should have findById and throw exception 404 not found if it doesn't exist and not doing this additional boolean check

Reply:
I thought that's validate it means update all exist dayTodo for synchronize back and front. If I throw exception we don't update any order number. Ok I will change this method.